### PR TITLE
Resolve some TODO items

### PR DESCRIPTION
**Pull request by bjss-colin**

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/CataloguePrice.cs [hunk 1]
```diff
@@ -17,7 +17,6 @@ public CataloguePrice()
 
         public CatalogueItemId CatalogueItemId { get; set; }
 
-        // TODO: remove
```
Comment (bjss-colin) [inline src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/CataloguePrice.cs:20]: ID is used to build the relationship. Comment removed instead.

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Ordering/Models/Order.cs [hunk 2]
```diff
@@ -21,12 +21,10 @@ public Order()
 
         public string Description { get; set; }
 
-        // TODO: remove
         public int OrderingPartyId { get; set; }
```
Comment (bjss-colin) [inline src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Ordering/Models/Order.cs:24]: ID is used to build the relationship. Comment removed instead.

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Ordering/Models/Order.cs [hunk 3]
```diff
@@ -21,12 +21,10 @@ public Order()
...
 
         public int? OrderingPartyContactId { get; set; }
 
-        // TODO: remove
         public int? SupplierId { get; set; }
```
Comment (bjss-colin) [inline src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Ordering/Models/Order.cs:28]: ID is used in many places.

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Solutions/Controllers/SolutionsControllerTests.cs [hunk 4]
```diff
@@ -173,8 +173,6 @@ public static void Constructors_VerifyGuardClauses()
         {
             var catalogueItem = solution.CatalogueItem;
             solution.ClientApplication = JsonSerializer.Serialize(clientApplication);
-
-            // TODO: add AutoFixture customization (exclude certain base properties)
```
Comment (bjss-colin) [inline tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Solutions/Controllers/SolutionsControllerTests.cs:177]: Comment removed as not being worth the effort. This is only used in a single place (here).

**Review by PhysicsJon — APPROVED** [review]
Looks sensible enough

**Pull request merged**